samples/tab-stage-view/nodejs/.env

```diff
@@ -1,3 +1,3 @@
-MicrosoftAppId=""
-MicrosoftAppPassword=""
-BaseUrl=""
+MicrosoftAppId="<<MICROSOFT-APP-ID>>"
+MicrosoftAppPassword="<<MICROSOFT-APP-SECRET>>"
```

Comment:
<<MICROSOFT-APP-SECRET>>" This should be consistent in all the samples.
Other places, we are using MICROSOFT-APP-PASSWORD